### PR TITLE
Remove -*archive command, cpio -> initcpio

### PR DESCRIPTION
**Pull request by andrewchambers**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -45,8 +52,12 @@ test: all
 install: all
 	mkdir -p $(DESTDIR)$(BINDIR)
 	cp $(BIN) $(DESTDIR)$(BINDIR)
-	ln -sf fspec-archive $(DESTDIR)$(BINDIR)/fspec-tar
-	ln -sf fspec-archive $(DESTDIR)$(BINDIR)/fspec-cpio
+	cd $(DESTDIR)$(BINDIR) \
+	  && ln -f fspec-{archive,tar} \
```
Comment (michaelforney) [inline Makefile]: This brace expansion syntax is non-POSIX. I'd just  write out the full commands.

File: src/fspec-archive.c [hunk 2]
```diff
@@ -61,40 +61,27 @@ main(int argc, char **argv)
...
-
-    if (!archive_format(a) && prog) {
-        if (strcmp(prog, "fspec-tar") == 0)
-            archive_write_set_format_pax_restricted(a);
-        else if (strcmp(prog, "fspec-cpio") == 0)
-            archive_write_set_format_cpio_newc(a);
-    }
-    if (!archive_format(a))
-        errx(1, "archive format could not be inferred, and was not specified explicitly");
+    if (strcmp(prog, "fspec-tar") == 0)
```
Comment (michaelforney) [inline src/fspec-archive.c]: I think we need to keep the `if (prog)` check. It is possible to exec a program with `argv[0] == NULL`.

File: src/fspec-archive.c [hunk 3]
```diff
@@ -61,45 +61,46 @@ main(int argc, char **argv)
...
+    if (strcmp(prog, "fspec-tar") == 0) {
+        archive_write_set_format_pax_restricted(a);
+    } else if (strcmp(prog, "fspec-initcpio") == 0) {
+        archive_write_set_format_cpio_newc(a);
+    } else if (strcmp(prog, "fspec-iso") == 0) {
+        archive_write_set_format_iso9660(a);
+    } else if (strcmp(prog, "fspec-isolinux") == 0) {
+        archive_write_set_format_iso9660(a);
+        if ( archive_write_set_option(a, "iso9660", "boot", "isolinux/isolinux.bin") != ARCHIVE_OK
+          || archive_write_set_option(a, "iso9660", "boot-catalog", "isolinux/boot.cat") != ARCHIVE_OK
```
Comment (michaelforney) [inline src/fspec-archive.c:86]: Where do these files come from? Do you have to run `fspec-isolinux` from some particular directory?

Comment (andrewchambers) [reply src/fspec-archive.c:86]: Nah, they just need to be in your fspec, they come from your syslinux install so its just up to the user. These are just paths embedded into the cd metadata,

**Pull request closed without merging**